presto-memory-context/src/main/java/com/facebook/presto/memory/context/AbstractAggregatedMemoryContext.java

```diff
@@ -32,7 +32,7 @@
     @GuardedBy("this")
     private long usedBytes;
     @GuardedBy("this")
-    private boolean closed;
+    protected boolean closed;
```

Comment:
i'd prefer `protected boolean isClosed()`, especially since this is `@GuardedBy`.
or `protected void checkNotClosed()`